products.d/slowroll.yaml

```diff
@@ -0,0 +1,143 @@
+id: Slowroll
+name: Slowroll
+# ------------------------------------------------------------------------------
+# WARNING: There are currently no translations at this time!
+# ------------------------------------------------------------------------------
+description: 'A slightly slower rolling release of openSUSE designed to update
+  less often than Tumbleweed but more often than Leap without forcing users to
+  choose between "stable" and newer packages'
+icon: Slowroll.svg
```

Comment:
You might need to add the icon to `web/src/assets/products`. In the future, the icon will be placed next to this definition, but we are not there yet.

Reply:
@imobachgs looks like I'm not allowed to add images only code, but it would be the file from https://en.opensuse.org/File:Slowroll_logo_by_pprmint_text.svg since it was uploaded by @bmwiedemann and appears to be official

Reply:
Maybe the color of the logo will change later to create some distinction from the other openSUSE distributions. We can update images when that happens.